tests/coalaJSONTest.py

```diff
@@ -113,7 +113,7 @@ def test_version_conflict_in_collecting_bears(self, import_fn, _):
             import_fn.side_effect = VersionConflict('msg1', 'msg2')
             retval, stdout, stderr = execute_coala(
                 coala.main, 'coala', '--json', '-B')
-            self.assertEqual(retval, TEST_BEARS_COUNT)
+            self.assertEqual(retval, 13)
```

Comment:
Because `13` here is not a bear count but an exit code https://github.com/coala/coala/blob/master/coalib/misc/Exceptions.py#L21

I mistakenly change this in prev PR because the number is same :(